bundles/admin/admin-permissions/view/LayerRightsTable.jsx

```diff
@@ -97,6 +97,7 @@ export const LayerRightsTable = ThemeConsumer(({ theme, controller, state }) =>
             dataIndex: 'name',
             sorter: getSorterFor('name'),
             render: (title, item) => {
+                const layer = Oskari.getSandbox().getService('Oskari.mapframework.service.MapLayerService').findMapLayer(item.id);
```

Comment:
Get the reference to the MapLayerService before the render. The reference won't change between renders and it can be reused. Probably doesn't affect much but it reduces lookups as render is called multiple times.